src/frontend/src/repositories/anchorMetadata.test.ts

```diff
@@ -0,0 +1,223 @@
+import { MetadataMapV2 } from "$generated/internet_identity_types";
```

Comment:
I think it would be good to have a test to ensure that the repository leaves additional data unchanged in `rawMetadata`.

Reply:
I had a test for this: `"IdentityMetadataRepository commits additional metadata to canister after update"`.

I rephrased the description to use "additional", maybe that was the confusing part.